islandora_ordered_zip_newspaper_batch.module

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @file
+ * Module used to handle newspaper batches through the UI and drush.
+ */
+
+/**
+ * Implements hook_menu().
+ */
+function islandora_ordered_zip_newspaper_batch_menu() {
+  $items = array();
+
+  $items['islandora/object/%islandora_object/manage/newspaper_batch'] = array(
```

Comment:
This collides with the [base newspaper batch](https://github.com/Islandora/islandora_newspaper_batch/blob/7.x/islandora_newspaper_batch.module#L13-L21).